02-microarray/00-intro-to-microarray.Rmd

```diff
@@ -1,8 +1,82 @@
+
+### Microarray data **strengths**:  
+
+- Microarray is generally less expensive than RNA-seq - you can afford more replicates and get higher statistical power [@Tarca2006]. 
+- Microarray often has a faster turn-around than RNA-seq [@LCSciences2014].
+
+### Microarray data **limitations**:  
+
+- If a transcript doesn't have a probe designed to it on a microarray, it won't be measured; microarray can't be used for transcript discovery [@Mantione2014]. 
+- A chip's probe designs are only as up to date as the genome annotation at the time it was designed [@Mantione2014].
+- As is true for all techniques that involve nucleotide hybiridization (RNA-seq too); microarray probes come with some biases depending on their nucleotide sequence composition (like GC bias). 
+
+Refine.bio drops outdated probes based on [Brainarray’s annotation packages](http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/) and uses [SCAN](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3508193/pdf/nihms401888.pdf) normalization methods prior to your downloads to help address these probe nucleotide composition biases [@Dai2005; @Piccolo2012].
+
+## About quantile normalization 
+
+Microarray chips are generally experimentally processed in groups of chips - this can lead to [experimental batch effects](https://en.wikipedia.org/wiki/Batch_effect#:~:text=In%20molecular%20biology%2C%20a%20batch,of%20interest%20in%20an%20experiment). 
+To minimize this, all refine.bio microarray data downloads come [quantile-normalized](https://en.wikipedia.org/wiki/Quantile_normalization) which enables more confident comparisons of expression levels among experiments.
+Different microarray chips, are also a type of batch effect, but quantile normalizations allows us to compare data from different chips that to a limited degree if we proceed with caution. 
+See the refine.bio docs for more about the microarray processing steps, including the [quantile normalization](http://docs.refine.bio/en/latest/main_text.html#quantile-normalization).
+
+## More resources on microarray technology: 
+
+- [Getting started in gene expression microarray analysis](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000543) [@Slonim2009].
+- [Microarray and its applications](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3467903/) [@Govindarajan2012].
+- [An tutorial review of microarray data analysis](http://www.ub.edu/stat/docencia/bioinformatica/microarrays/ADM/slides/A_Tutorial_Review_of_Microarray_data_Analysis_17-06-08.pdf) [@Sanchez2008].
+- [Analysis of microarray experiments of gene expression profiling](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2435252/) [@Tarca2006].
+- [Some nice summary slides about microarray data analysis from Emory](http://web1.sph.emory.edu/users/hwu30/teaching/bioc/GE1.pdf) [@intro-microarray-slides].
+
+### Why doesn't the gene I care about show up in this microarray dataset?
```

Comment:
I think there are a few things that should be addressed here: 

One is the case where people have a single experiment that they have downloaded, which you have partly addressed here with the Brainarray discussion. But some arrays do not have Brainarray annotations (all plants, but also many human arrays, according to https://github.com/AlexsLemonade/refinebio/blob/dev/config/supported_microarray_platforms.csv so this does not universally apply. The simple explanation of "it wasn't on the chip" should probably be added.

The other case is where people are using refine.bio to combine across experiments that may have been done on different chips with different gene content. This results in genes not present on both chips being dropped, and this should be discussed as well. https://docs.refine.bio/en/latest/main_text.html#aggregations

Reply:
I guess I haven't been thinking about the aggregate cases since none of our examples or planned future examples use the aggregate option.

Reply:
Okay, I've tried to better address both of these points.